src/Microsoft.AspNet.Mvc.Common/project.json

```diff
@@ -8,9 +8,7 @@
         "dnx451": { },
         "dnxcore50": {
             "dependencies": {
-		"System.ComponentModel.TypeConverter": "4.0.0-beta-*",
-                "System.Reflection.Extensions": "4.0.0-beta-*",
-                "System.Text.Encoding.Extensions": "4.0.10-beta-*"
+                "System.Runtime": "4.0.20-beta-*"
```

Comment:
Should we print a warning \ make it an error to have any dependencies in a shared source project if this is the intended behavior?